Applications/HexEditor/GHexEditor.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+#include <AK/Function.h>
+#include <AK/HashMap.h>
+#include <AK/NonnullOwnPtrVector.h>
+#include <AK/NonnullRefPtrVector.h>
+#include <AK/StdLibExtras.h>
+#include <LibDraw/TextAlignment.h>
+#include <LibGUI/GScrollableWidget.h>
+
+class GHexEditor : public GScrollableWidget {
```

Comment:
The `G` prefix is generally used for classes in LibGUI. Since this class is outside LibGUI, you can just call it `HexEditor` or something like that instead.